test/Microsoft.AspNet.Mvc.Core.Test/ParameterBinding/ControllerActionArgumentBinderTests.cs

```diff
@@ -361,7 +361,7 @@ public class ControllerActionArgumentBinderTests
             binder
                 .Setup(b => b.BindModelAsync(It.IsAny<ModelBindingContext>()))
                 .Returns(Task.FromResult(
-                    result: new ModelBindingResult(model: null, key: string.Empty, isModelSet: true)));
+                    result: new ModelBindingResult(model: null, key: string.Empty, isModelSet: false)));
```

Comment:
See `DefaultControllerActionArgumentBinder`@130 - there's code in place to specifically handle this scenario, and this is a test written specifically to target that code.